### PR TITLE
Allos disabling webhelper

### PR DESCRIPTION
**Pull request by ebassi**
https://phabricator.endlessm.com/T15749

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -226,18 +226,29 @@ AC_SUBST([JASMINE_REPORT_ARGUMENT])
...
 
 # Check installed GIRs for webhelper JS module
 EOS_CHECK_GJS_GIR([GLib], [2.0])
-EOS_CHECK_GJS_GIR([WebKit], [3.0])
-EOS_CHECK_GJS_GIR([WebKit2], [4.0])
+
+# Webhelper checks
+AC_ARG_ENABLE([webhelper],
+              [AS_HELP_STRING([--disable-webhelper], [Disable webhelper build])],
+              [],
```
Comment (ptomato) [inline configure.ac]: nitpick: The Autotools Mythbuster recommends as a best practice to leave out the optional arguments to `AC_ARG_ENABLE` and, for a default-enabled argument, make all the checks (`AS_IF`, `AM_CONDITIONAL`) work like `test "x$enable_webhelper" != xno`.

**Comment by ebassi** [comment]
I actually think we should cherry pick back the commit you wrote for the eos-programming app flatpak, to be honest. Looking at it, it's definitely more thorough than my hatchet job, and we already know it's working.

**Comment by ptomato** [comment]
Go ahead, though it won't apply cleanly - the eos2.6 branch _also_ had a separate webhelper for WebKit2-3.0 (as well as WebKit-3.0 and WebKit2-4.0 which are still on the master branch)

**Comment by ebassi** [comment]
Updated with a proper cherry pick.

**Comment by ptomato** [comment]
Yep, this works fine. I did discover that `make dist` won't work if you haven't got `--enable-webhelper`, but I think that's to be expected.

**Pull request merged**